### PR TITLE
[DQL3-73] Update data_last_updated

### PR DESCRIPTION
**Pull request by tino097**
Create command to update `data_last_updated` if not set

### CODE REVIEW

**Comment by ThrawnCA** [comment]
Failing lint.

Also, can the commands please be grouped? So instead of `paster update_datasets` and `paster demote_publishers` and so forth, it would be something like `paster data_qld <command>`, with just one entry in `setup.py` and all commands in a single class. That has several advantages:

- Less duplication in `setup.py`
- Less duplication in the command module
- Easier to produce help documentation for every available command
- Less chance of a name clash with another extension's commands

Edit: https://github.com/qld-gov-au/ckanext-validation/blob/master/ckanext/validation/commands.py has an example of how to put multiple commands in one class, with the `command` function then checking its arguments to see which function to call.

**Review by ThrawnCA — CHANGES_REQUESTED** [review]
Could use a restructure.

**Review by ThrawnCA — CHANGES_REQUESTED** [review]
Still failing lint.

File: ckanext/data_qld/commands.py [hunk 1]
```diff
@@ -10,54 +12,78 @@
...
 
 
-class MigrateExtras(CkanCommand):
-    """Migrates legacy field values that were added as free extras to datasets to their schema counterparts.
+class DataQld(CkanCommand):
+    """
+    Data Qld custom commands
+
+    Usage:
+        paster data_qld
```
Comment (ThrawnCA) [inline ckanext/data_qld/commands.py:20]: Would be good to document the possible commands here, but as it's for internal use, not critical.

**Review by ThrawnCA — APPROVED** [review]

**Review by ThrawnCA — APPROVED** [review]

File: ckanext/data_qld/commands.py (hunk 1 above)
Comment (tino097) [reply ckanext/data_qld/commands.py:20]: i guess we need to add all the command and i could add few lines for the usage

**Review by ThrawnCA — APPROVED** [review]

**Comment by tino097** [comment]
This PR is part of #61

**Pull request closed without merging**